### DISCUSSION TITLE
Support x-codeSamples option?

### DISCUSSION THREAD

**Discussion by tmcw** [scalar/scalar#2026] [Ideas]
We're looking into using Stainless to generate an SDK, and its way of integration with documentation sites is via x-codeSamples - which looks like it also works with Mintlify and Redocly ( https://mintlify.com/docs/api-playground/openapi/advanced-features https://redocly.com/docs/api-reference-docs/specification-extensions/x-code-samples/ ) - https://app.stainlessapi.com/docs/guides/integrate-docs

Could scalar support this? The workflow seems like it'll be pretty convoluted regardless of how we build it because we'll basically have to generate the SDK with OpenAPI, and then generate OpenAPI using the SDK thing, then generate the docs with that 😆

**Comment by hanspagel** [comment]
Not documented, but supported! :)

https://sandbox.scalar.com/e/KZYm6
